Product/Production/Services/AuditRepositoryCore/src/main/java/gov/hhs/fha/nhinc/auditrepository/nhinc/AuditRepositoryOrchImpl.java

```diff
@@ -116,7 +116,9 @@ public AcknowledgementType logAudit(LogEventSecureRequestType mess, AssertionTyp
             if (activeParticipant != null) {
                 eventUserId = activeParticipant.getUserID();
                 if (eventUserId != null && !eventUserId.equals("")) {
-                    auditRec.setUserId(eventUserId);
+                    /* this value is temporary set to empty strings and due to length of characters are exceeding
+                     more than 100*/
+                    auditRec.setUserId("");
```

Comment:
Is there a limitation of 100 chars in the schema?  Is it better to set blank, than to truncate?